src/views/EdgeApplicationsFunctions/FormFields/FormFieldsEdgeApplicationsFunctions.vue

```diff
@@ -44,23 +43,24 @@
   <FormHorizontal
     :isDrawer="true"
     title="General"
-    description="Instantiate an edge function within your edge application."
+    description="Instantiate a serverless function created with Edge Functions within the edge application. Use Rules Engine to activate the function."
   >
     <template #inputs>
       <div class="flex flex-col sm:max-w-lg w-full gap-2">
         <FieldText
           label="Name *"
           name="name"
           v-model="name"
+          description="Give a unique and descriptive name to identify the function instance."
         />
       </div>
     </template>
   </FormHorizontal>
 
   <FormHorizontal
     :isDrawer="true"
-    title="Function Instanced"
-    description="Description. Informar que para criar novas functions para serem instanciadas, o usuário precisa ir em Edge Functions."
+    title="Edge Function"
+    description="Select the edge function and pass the required arguments. The function can't be modified at this stage."
```

Comment:
sugestão que pensei pro firewall:

"Select an existing edge function and customize the arguments. You can only instantiate edge functions previously created in the Edge Functions module."